code/modules/shuttle/dropship_hijack.dm

```diff
@@ -116,8 +118,10 @@
 		remaining_crash_sites -= target_ship_section
 		var/new_target_ship_section = pick(remaining_crash_sites)
 		var/turf/target = get_crashsite_turf(new_target_ship_section)
-		crash_site.Move(target)
 		marine_announcement("A hostile aircraft on course for the [target_ship_section] has been successfully deterred.", "IX-50 MGAD System")
+		crash_site.x = target.x + HIJACK_CRASH_SITE_OFFSET_X
+		crash_site.y = target.y + HIJACK_CRASH_SITE_OFFSET_Y
+		crash_site.z = target.z
```

Comment:
there shouldn't be any direct loc / coordinates setting basically ever, use Move() or forceMove()

for the above one it can even just be passed as constructor argument

Reply:
No real point to set up a constructor if the crash site is attempted to be reused so then we need different code for that case. Should just be the same method for both procs.